pkg/proxy/proxy_test.go

```
@@ -0,0 +1,96 @@
+			t.Errorf("Failed to create proxy: %v", err)
+			ch <- nil
+			return
+		}
+		ch <- proxy
+	}()
+
+	host := server.Addr().String()
+	port := host[strings.IndexByte(host, ':')+1:]
+
+	request := "GET http://localhost:" + port + "/ HTTP/1.1\r\nHost: localhost\r\n\r\n"
+	clientConn.Write([]byte(request))
```

golangci-lint (1.62.2)

Error return value of `clientConn.Write` is not checked

(errcheck)